avatar.css

```diff
@@ -0,0 +1,76 @@
+@import url(components.css);
+
+h2
+{
+    text-align: center;
+}
+main{
+    text-align: center;
+   
+    
+}
+
+.XXL
```

Comment:
Be consistent with the class names I could spot both small and upper case. I would suggest you to use small case for naming.

Reply:
Sure ,Next Time I'll keep note on that!